docs/manual/annotating-libraries.tex

```diff
@@ -568,10 +568,10 @@ \subsubsection{If you do not have access to the Java source code\label{stub-crea
 \begin{enumerate}
 
 \item
-  Create a stub file by running the stub class generator.  (\<checker.jar> and \<javac.jar>
-  must be on your classpath.)
+  Create a stub file by running the stub class generator.  (\<checker.jar> must be on your classpath.)
 
 \begin{Verbatim}
+  export CLASSPATH=export CLASSPATH=$JSR308/checker-framework/checker/dist/checker.jar
   cd nullness-stub
   java org.checkerframework.framework.stub.StubGenerator java.lang.String > String.astub
```

Comment:
Can you remove setting the `CLASSPATH` variable (which incorrectly includes two `exports`) and change this to:

```
cd nullness-stub
java -cp $JSR308/checker-framework/checker/dist/checker.jar org.checkerframework.framework.stub.StubGenerator java.lang.String > String.astub
```

Using `-cp` instead of setting `CLASSPATH` is a more local change and achieves the same thing.

Reply:
Thanks for correcting it.